pkgs/tools/graphics/mangohud/default.nix

```diff
@@ -0,0 +1,74 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, fetchpatch
+, pkgconfig
+, ninja
+, meson
+, python3Packages
+, glslang
+, libglvnd
+, xorg
+, git
+, vulkan-loader
+, vulkan-headers
+, vulkan-tools
+, mesa
+, dbus
+}:
+
+
+stdenv.mkDerivation rec {
+  pname = "mangohud${lib.optionalString stdenv.is32bit "_32"}";
+  version = "0.4.1";
+
+  src = fetchFromGitHub {
+    owner = "flightlessmango";
+    repo = "MangoHud";
+    rev = "acf2d88fbcb7a7a47f957a87d20739c67049bd0d";
+    sha256 = "0qmprnxrvh8bkn6dnrk22v53am3flpixh046w7ach3pnfck1hpfb";
+  };
+
+  mesonFlags = [
+    "-Dappend_libdir_mangohud=false"
+    "-Dwith_xnvctrl=disabled"
```

Comment:
Was there a problem building with xnvctrl?

Reply:
I got it building now with this option enabled. I already included it in my nur package and will update it here later today:
https://github.com/ZerataX/nur-packages/blob/master/pkgs/MangoHud/default.nix#L34

Reply:
should now be resolved, though I don't have the hardware to test if this works with nvidia gpus.